### PR TITLE
Add Roadmap

### PR DESCRIPTION
**Pull request by cblanco**

### CODE REVIEW

File: documentation/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+-   Support more Context Broker actions (create, append, appendStrict, update, delete, replace, etc) and analyse other possible operations: batch operations, operations over subscriptions, operations over registrations.
+
+
+## Long term
+
+The following list of features are proposals regarding the longer-term evolution of the product even though development of these features has not yet been scheduled for a release in the near future. Please feel free to contact us if you wish to get involved in the implementation or influence the roadmap
+
+-   Support for geospatial analysis (geo-fencing, point-based actions, movement-based actions)
+
+-   Enhanced multi-tenancy support. Leverage different “Fiware-Service” and “Fiware-ServicePath” in rules
```
Comment (fgalan) [inline documentation/roadmap.md]: Better to use straitgh quotes `"..."` instead that `“...”`

File: documentation/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Perseo CEP Roadmap
+
+This product is an Incubated FIWARE Generic Enabler. If you would like to learn about the overall Roadmap of FIWARE, please check section "Roadmap" on the FIWARE Catalogue.
+
+## Introduction
+
+This section elaborates on proposed new features or tasks which are expected to be added to the product in the foreseeable future. There should be no assumption of a commitment to deliver these features on specific dates or in the order given. The development team will be doing their best to follow the proposed dates and priorities, but please bear in mind that plans to work on a given feature or task may be revised. All information is provided as a general guidelines only, and this section may be revised to provide newer information at any time.
+
```
Comment (fgalan) [inline documentation/roadmap.md]: You could add also the following disclaimer (https://github.com/telefonicaid/fiware-orion/blob/master/doc/roadmap.md#introduction):

* This section has been last updated in January 2019. Please take into account its content could be obsolete.
* Note we develop this software in Agile way, so development plan is continuously under review. Thus, this roadmap has to be understood as rough plan of features to be done along time which is fully valid only at the time of writing it. This roadmap has not be understood as a commitment on features and/or dates.
* Some of the roadmap items may be implemented by external community developers, out of the scope of GEi owners. Thus, the moment in which these features will be finalized cannot be assured.

Up to you :). If you prefer not adding it and keep it as it is now, then NTC on this comment.

File: documentation/roadmap.md (hunk 1 above)
Comment (cblanco) [reply documentation/roadmap.md]: Fixed in da7e1dc

File: documentation/roadmap.md (hunk 2 above)
Comment (cblanco) [reply documentation/roadmap.md]: Fixed in 8a43e97

**Comment by fgalan** [comment]
Some possible additions:

* Rule auto-naming on rule creation time (I think we discussed this in the past, although maybe is too minor to be included in a roadmap). Or it was already done? (not remember at this moment, sorry)
* Description API in OpenAPIv3 (I think we also mentioned this in the past)
* Rule templates (i.e. having a rule template that can be instanciated to create particular instances of the rule). This can be useful for rules with complex EPL, so the EPL is provide just one time (in the template), then each instance of rule based on that template only provides the specific information for that.

Just in case you want to consider adding them (maybe in the lont term stage).

**Comment by cblanco** [comment]
> Some possible additions:
> 
> * Rule auto-naming on rule creation time (I think we discussed this in the past, although maybe is too minor to be included in a roadmap). Or it was already done? (not remember at this moment, sorry)

I don't know if you are referring the feature added by PR #316. In the current roadmap file, you can see the description of this feature in the line number 21.

> * Description API in OpenAPIv3 (I think we also mentioned this in the past)

Ok. We have an initial version [here](https://github.com/Ficodes/perseo-fe/blob/aa095021e0edc45dbea664b99a11a55c9b49b389/documentation/API/perseo-api.yml). I will try to make a PR as soon as possible. 
Fixed in 7adf4c6

> * Rule templates (i.e. having a rule template that can be instanciated to create particular instances of the rule). This can be useful for rules with complex EPL, so the EPL is provide just one time (in the template), then each instance of rule based on that template only provides the specific information for that.
> 
> Just in case you want to consider adding them (maybe in the lont term stage).

Ok. Fixed in 7adf4c6

**Pull request merged**